academy/4-my-own-chain/server-side.md

```diff
@@ -1044,3 +1048,9 @@ You have created a small server that:
 This is an example of server-side scripts, one that can improve user experience.
 
 You can find the complete code [here](https://github.com/cosmos/academy-checkers-ui/tree/server-indexing).
+
+## Next up
+
+You now have a working Checkers GUI. This rather long preparatory effort was made in order to use recent packages. You are now ready to integrate it with the Checkers blockchain.
+
```

Comment:
Mmh, the server side thing should come after the GUI. And it is not a preparation for the GUI.